rules.ini

```diff
@@ -44,7 +44,16 @@ ChromeEngine[] = \.rpack$
 Clausewitz[] = (?:^|/)common/defines\.(?:lua|txt)$
 Clausewitz[] = ^(?:game/)?common/defines/00_defines\.txt$
 ClickTeamFusion = (?:^|/)mmfs2\.dll$
-Cocos2d = (?:^|/)libcocos2d\.dll$
+Cocos[] = (?:^|/)libcocos2d\.dll$
+Cocos[] = (?:^|/)Cocos2d\.lua$
+Cocos[] = (?:^|/)jsb-adapter/jsb-engine\.jsc?$
+Cocos[] = (?:^|/)jsb-adapter/engine-adapter\.jsc?$
+Cocos[] = (?:^|/)cocos2d-jsb\.[0-9a-z]+\.jsc?$
+Cocos[] = (?:^|/)cocos2d-jsb\.jsc?$
+Cocos[] = (?:^|/)assets/main/cc\.config\.json$
```

Comment:
Cocos Creator will generate `cc.config.json` instead of `cc.config.8384e.json` when `MD5 cache` in the build panel is disabled. 

<img width="784" alt="WeChat5df8317381abcb8140e1dcb9edb95677" src="https://user-images.githubusercontent.com/493372/196866613-4d62d955-4a2e-4c65-bde9-e2b24af34436.png">


We need to keep the both cases work, so it's better not to remove it.